src/Sylius/Behat/Page/Admin/Country/UpdatePage.php

```diff
@@ -117,7 +151,17 @@ public function removeProvinceName($provinceName)
         if ($this->isThereProvince($provinceName)) {
             $provinces = $this->getElement('provinces');
 
-            $item = $provinces->find('css', 'div[data-form-collection="item"] input[value="' . $provinceName . '"]')->getParent();
+            $item = $provinces->find('css', '[value="' . $provinceName . '"]')->getParent();
+            $item->fillField('Name', '');
+        }
+    }
+
+    public function removePostCode($postCodeName)
+    {
+        if ($this->isTherePostCode($postCodeName)) {
```

Comment:
Shouldn't it throw some `ElementNotFoundException` or sth if we want to remove not existing postcode?